sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
These minor indentation issues are really not a big deal. Each person has a different preference, and one just needs to get used to that other people might format things a bit differently, and not lose too much time discussing it. So I agree with other things that @toolforger proposed (like missing `.` or in other PRs missing space around operators like `==`), but do not lose time worrying about how to properly indent multiline command.